packages/eslint-plugin-pf-codemods/src/rules/v6/cardUpdatedClickableMarkup/card-updated-clickable-markup.ts

```diff
@@ -54,19 +53,23 @@ module.exports = {
               const selectableActionsValue = getAttributeValue(
                 context,
                 selectableActionsProp.value
-              );
+              ) as ObjectExpression["properties"];
```

Comment:
Here the casting is done because the actual `selectableActions` prop on `CardHeader` accepts only an object (CardHeaderSelectableActionsObject), so consumers shouldn't be able to pass anything else. I added a comment to make it clear.

The `getAttributeValue` should ultimately get to the ObjectExpression, or return undefined (e.g. if it was an Identifier imported from another file) in which case we don't do the fix

Reply:
But yeah on many other places, the casting was done due to the lack of typing of the `getAttributeValue` helper